src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,22 @@
+.catalog {
+  display: grid;
+  grid-template-columns: $card-width;
+  justify-content: center;
+  gap: 48px;
+
+  max-width: calcCatalogWidth($card-width, $catalog-item-horizontal-gap, 4);
+  margin: 0 auto;
```

Comment:
Remove this.

Reply:
Yes, some redundant staff from previous task. Removed it.